.travis.yml

```diff
@@ -34,4 +31,5 @@ script:
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_snmp_information
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_users
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_traceroute
+- nosetests -v TestNXOSDriver:TestNXOSDriver_bgp_neighbors
```

Comment:
@dbarrosop - these tests use setUp, and not setUpClass.  Do you think that people will find that confusing?

I've actually never seen test cases use the class as the basis for test instance data.  I've used setUpClass and similar to create data in exceptional cases where the set up took a _very long_ time, like with databases.  Can you recall the technical reason for choosing setUpClass, rather than just plain setUp?

I'm ok with moving it over, I've tried it locally and it works.  Will do after you consider the above point and give the go-ahead.